po/extra/fr.po

```diff
@@ -0,0 +1,61 @@
+# French translations for extra package.
+# Copyright (C) 2020 THE extra'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# Nathan Bonnemains (@NathanBnm), 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-06-01 12:24+0200\n"
+"PO-Revision-Date: 2020-06-01 10:15+0200\n"
+"Last-Translator: Nathan Bonnemains (@NathanBnm)\n"
+"Language-Team: Français\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: data/com.github.rajkumaar23.badgie.desktop.in:3
+#: data/com.github.rajkumaar23.badgie.appdata.xml.in:7
+msgid "Badgie"
+msgstr "Badgie"
+
+#: data/com.github.rajkumaar23.badgie.desktop.in:4
+msgid "Badge Generator"
+msgstr "Générateur de badges"
+
+#: data/com.github.rajkumaar23.badgie.desktop.in:5
+msgid ""
+"Generate realtime badges from PlayStore data for your android app README"
+msgstr ""
+"Générez des badges en temps réel depuis les données du PlayStore pour le "
```

Comment:
This translates to `Generate badges in real time from PlayStore data for the README file from your Android application`. But it should rather be `..for your Android application`

Reply:
Didn't notice that but the end of the sentence is missing. I'll fix it asap.

Reply:
> This translates to `Generate badges in real time from PlayStore data for the README file from your Android application`. But it should rather be `..for your Android application`

@rajkumaar23 just checked and the translation is fine.

The French to English translation you made is incorrect 😅

Reply:
But that's what is shown [here](https://translate.google.co.in/#view=home&op=translate&sl=fr&tl=en&text=G%C3%A9n%C3%A9rez%20des%20badges%20en%20temps%20r%C3%A9el%20depuis%20les%20donn%C3%A9es%20du%20PlayStore%20pour%20le%20%22%0A%22fichier%20README%20de%20votre%20application%20Android) :thinking:

Reply:
> But that's what is shown [here](https://translate.google.co.in/#view=home&op=translate&sl=fr&tl=en&text=G%C3%A9n%C3%A9rez%20des%20badges%20en%20temps%20r%C3%A9el%20depuis%20les%20donn%C3%A9es%20du%20PlayStore%20pour%20le%20%22%0A%22fichier%20README%20de%20votre%20application%20Android) :thinking: 

I'm a native French speaker and Google Translate is not 100% reliable. I know what I'm doing 😅

Reply:
Haha. That's okay. Anyways, thanks for the help! I'm merging :)

Reply:
Well, Google Translate translation is okay if you remove the extra quotes and line return.

See [here](https://translate.google.co.in/#view=home&op=translate&sl=fr&tl=en&text=G%C3%A9n%C3%A9rez%20des%20badges%20en%20temps%20r%C3%A9el%20depuis%20les%20donn%C3%A9es%20du%20PlayStore%20pour%20le%20fichier%20README%20de%20votre%20application%20Android)

Reply:
@rajkumaar23 otherwise I recommend you to use Deepl instead of Google Translate when needed because it's more reliable and you can click on the words in the translated block to get alternatives and change the translation.

Reply:
Alright. Thanks!